tapdb/sqlc/migrations/000024_universe_optimization_indexes_queries.up.sql

```diff
@@ -0,0 +1,53 @@
+-- Core universe_roots indexes
```

Comment:
nit: we generally prefer all comments to be full sentences (including punctuation).